docs/source/getting-started/installation.rst

```diff
@@ -46,6 +46,18 @@ If you're interested in using our visualizations, you can optionally install the
 
 We use Vue.js as our frontend web framework, and ``npm`` as our package manager.
 
+Visualiation API Calls
+----------------------
+On Ubuntu and other Linux Flavors: If you want to use the new Augur API Calls that generate downloadable graphics developed in the https://github.com/chaoss/augur-community-reports repository you need to install the Chromium Chromedriver at the system level. 
+
+-  `sudo apt install chromium-chromedriver`
+
+This dependency exists because the Bokeh libraries we use for these APIs require a web browser engine. 
+
+For Mac OSX you can use: 
+
+-  `brew install chromedriver`
+
```

Comment:
This documentation is not correctly formatted and has a few grammatical and spelling errors, as I've noted below. Please fix them.

- "Visualization" is misspelled in the section title
- "Calls" should not be capitalized in the section title
- "Flavors" should not be capitalized
- "If" should not be capitalized
- Please use an embedded hyperlink target for `https://github.com/chaoss/augur-community-reports`
- Please add a comma after the word "repository"
- "ChromeDriver" is improperly capitalized
- `sudo apt install chromium-chromedriver` is not correctly formatted. Please use a `code-block` directive.
- `brew install chromedriver` is not correctly formatted. Please use a `code-block` directive.
-  The sentence `This dependency exists because the Bokeh libraries we use for these APIs require a web browser engine.` should be included somewhere in the first paragraph.

Reply:
@sgoggins

Reply:
fixed.